data/human/human missions.txt

```diff
@@ -7454,3 +7454,216 @@ mission "Adelita 5"
 			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumbles as she looks through a stack of folders on her datapad. "Now we have to watch it over and over again to edit it into something presentable."`
 			`	As they are leaving, Ale hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you. We really couldn't have done it without you," she tells you with a bittersweet smile. "Of course, you'll be one of the first to know when it's done."`
 			`	She starts off to join River and Bunker out in the spaceport before turning around one last time in the doorway. "We might be in need of you for some future projects too, Captain <last>. But goodbye for now. Stay safe on the star lanes!"`
+
+
+
+mission "Syndicate Business 1"
+	name "Rescue Kidnapped Man in <waypoints>"
```

Comment:
The capitalization change here is definitely a good idea. I'm not sure about the addition of `a`, though, since this should read more like a headline than a sentence.